### ISSUE TITLE
AFC-660 Test new GUI for visually impaired readers

### ISSUE THREAD

**Issue by FredrikArvas** [vertelab/odoo-af#263]
**Userstory:** 
As a blind user, I want to be able to use the Odoo-system in a simplified version

![image](https://user-images.githubusercontent.com/9428370/96464330-f8ac5480-1227-11eb-9ca3-a9ac1fa30b37.png)

Try to display the Jobbseeker-contact card in the web-portal in parallel to the normal interface. 

Desired functions: 
Search-view (similar to the "administrative officer" view)
Form-View (simiar to the Jobbseeker contact form)

**Screen-shots before:**

**Screen-shots after:**

**Module name:**
https://github.com/vertelab/odoo-af/tree/12.0-AFC-660/af-a11y

**Module type:** 
?? Application/Property/Resource/Glue module [see https://github.com/vertelab/odoo-af/issues/178 for more info]

**Test-steps:**
Login with "blind"-web-browser

**Test-data:**

**Branch name:** 
12.0-AFC-613-Web-gui_jobbseeker

**Description:** 
This module adds a webinterface to the jobbseeker contact-info

**Estimate:** 
Estimated Hours: 
Estimated Date: 
Actual Hours:

**Comment by FredrikArvas** [comment]
Hi Miracle, plase use this module in this branch instead of creating a new one.
https://github.com/vertelab/odoo-af/tree/12.0-AFC-660/af-a11y
Konrad @Viltersten prepared this for us.
/Fredrik

**Comment by Viltersten** [comment]
I missed to push the cleaned-up version of the module due to connection issues. It should be corrected now.

**Comment by iammiracle** [comment]
Hi Fredrick,

I have come up with something, kindly see:

**Contact is added to the link** 

![image](https://user-images.githubusercontent.com/7355805/96716360-d6881300-139c-11eb-8edd-bfddd863d047.png)

**Shows all Contacts**

![image](https://user-images.githubusercontent.com/7355805/96716462-f7e8ff00-139c-11eb-8653-f02ac6ed5420.png)

**Each Contact Details**

![image](https://user-images.githubusercontent.com/7355805/96716523-0c2cfc00-139d-11eb-9e47-7983a66204c9.png)

[PR created](https://github.com/vertelab/odoo-af/pull/279)

**Comment by FredrikArvas** [comment]
Please close this PR https://github.com/vertelab/odoo-af/pull/279 and create a new one here
https://github.com/vertelab/odoo-web/tree/Dev-12.0-Fenix-Sprint-02 with a new name of the module: web_backend_theme_af
Thanks

**Comment by FredrikArvas** [comment]
We will not go on with this direction.
